README.md

```diff
@@ -150,57 +152,68 @@ You can check out the recorded demo to see how the navigator works.
 ## Publishing to the Chrome Web Store
 
 1. Prepare the code and the `master` branch.
-   1. Increase the `version` number in the [manifest](src/manifest.json#L6) file and push changes to the `master`
-      branch.
+   1. Update the `version` number in the [manifest](src/manifest.json#L3) and [package](package.json#L3) files.
+      1. The project follows semantic versioning, i.e. the version is in the format "{MAJOR.MINOR.PATCH}[-dev]". For
+         example, 1.0.15 and 1.1.9-dev.  
+         (Refer to the [semantic versioning](https://semver.org/) standard documentation for more details)
+      2. If the release is a **full release**, use one of the following commands.
```

Comment:
I have tested the release updating, in general it's very nice.  I would argue that the default should not be a full major release.  Most often you will be creating dev patch releases, IMO that should be the default.  You should have to indicate that the release is "full".

Is it possible to use flags rather than using multiple colons in the task name?  That seems preferable.